README.md

```diff
@@ -46,8 +46,9 @@ and how it advertises itself to a Cryostat server instance. Required properties
 - [ ] `cryostat.agent.registration.retry-ms` [`long`]: the duration in milliseconds between attempts to register with the Cryostat server. Default `5000`.
 - [ ] `cryostat.agent.harvester.period-ms` [`long`]: the length of time between JFR collections and pushes by the harvester. This also controls the maximum age of data stored in the buffer for the harvester's managed Flight Recording. Every `period-ms` the harvester will upload a JFR binary file to the `cryostat.agent.baseuri` archives. Default `-1`, which indicates no harvesting will be performed.
 - [ ] `cryostat.agent.harvester.template` [`String`]: the name of the `.jfc` event template configuration to use for the harvester's managed Flight Recording. Default `default`, the continous monitoring event template.
-- [ ] `cryostat.agent.harvester.exit.max-age-ms [`long`]`: the JFR `maxage` setting, specified in milliseconds, to apply to recording data uploaded to the Cryostat server when the JVM this Agent instance is attached to exits. This ensures that tail-end data is captured between the last periodic push and the application exit. Exit uploads only occur when the application receives `SIGINT`/`SIGTERM` from the operating system or container platform.
-- [ ] `cryostat.agent.harvester.exit.max-size-b [`long`]`: the JFR `maxsize` setting, specified in bytes, to apply to exit uploads as described above.
+- [ ] `cryostat.agent.harvester.max-files` [`String`]: the maximum number of files that the agent will keep within the target. The agent will periodically prune, in a FIFO manner, the oldest JFR files within the target's storage, while the number of saved recordings is greater than the maximum file limit. Default `10`.
```

Comment:
This description implies that the recording storage is local to the agent somehow. I think it should be rephrased to be more clear that the configuration still results in the harvester pushing the files over the network, but this setting is used to set a policy on the storage backend for how many of these pushes to retain.